index.html

```diff
@@ -4,20 +4,19 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
-  <title>My Portfolio</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Karel van Oordt's Personal Portfolio</title>
```

Comment:
- [ ] For the page title, it is better to use a more meaningful and easier name such as Oordt's portfolio